samples/matlab_experimental/rankine.m

```diff
@@ -10,35 +10,39 @@
 
 help rankine
 
-% Initialize parameters
+%% Initialize parameters
 eta_pump = 0.6;
 eta_turbine = 0.8;
 p_max = 8.0 * OneAtm;
 t1 = 300.0;
 
-% create an object representing water
+%% create an object representing water
 w = Water;
 
+%%
 % start with saturated liquid water at t1
 basis = 'mass';
 w.TQ = {t1, 0};
 h1 = w.H;
 p1 = w.P;
 
+%%
 % pump it to p2
```

Comment:
In most cases you're adding the cell marker to the same line as the rest of the comment for a single line comment. Here and elsewhere you're adding a new line. Can it be consistent please?